database/tables/questions.pg

```diff
@@ -9,8 +9,10 @@ columns
     external_grading_files: text[] default ARRAY[]::text[]
     external_grading_image: text
     external_grading_timeout: integer
+    filename: text
```

Comment:
I think we should give this column a more descriptive name, otherwise it might be very unclear what the "question filename" means.

```suggestion
    thumbnail_filename: text
```